app/presenters/result_set_presenter.rb

```diff
@@ -78,6 +90,22 @@ def signup_links
 
   attr_reader :view_context
 
+  def no_top_results(documents)
+    documents.length < 2 ||
+      !sort_option ||
+      !sort_option["key"].eql?("-relevance") ||
+      finder.slug != "/find-eu-exit-guidance-business"
```

Comment:
It's possible that we might want to roll this pattern out to other finders in the (near) future, so perhaps we should separate out the slug check to either another method, or check if the finder slug is in an allowed list.